### DISCUSSION TITLE
Can you add ~(tilda) key to the list of

### DISCUSSION THREAD

**Discussion by 4Sol** [WarmUpTill/SceneSwitcher#959] [Ideas]
Can you add ~(tilda) key to the hotkey macro list?
This is a very important key and I can't bind it, because there are no such key in the list.
Thank you.

**Comment by WarmUpTill** [comment]
I added the `~` key to the hotkey action in this build:
https://github.com/WarmUpTill/SceneSwitcher/actions/runs/7400941223?pr=967

![image](https://github.com/WarmUpTill/SceneSwitcher/assets/19472752/4e43316d-7b49-4b9f-982b-9c7c0d77f018)

Unfortunately I could not really test it since the `~` key is not a separate key on my keyboard layout and requires pressing a modifier key.
So I don't really know if that works.

Which language keyboard layout has the `~` key as its "own key"?

**Reply by 4Sol** [reply]
I will test it later on, thank you for the update.
**Fn +escape** or **FN + Shift + escape** is usually where they put tilda key. Sometimes it's it's a another number or the back space or tab.
I think it's due do the keyboard size matter. Allsized keyboards should all have this key.
